src/DynamoCoreWpf/Controls/NotificationsControl.xaml.cs

```diff
@@ -1,47 +1,13 @@
-﻿using System.Windows;
-using System.Windows.Controls;
-using System.Windows.Input;
-
-using Dynamo.Controls;
-using Dynamo.Utilities;
-using Dynamo.ViewModels;
-using Dynamo.Wpf.ViewModels.Core;
-
-namespace Dynamo.Wpf.Controls
+﻿namespace Dynamo.Wpf.Controls
 {
     /// <summary>
     /// Interaction logic for NotificationsControl.xaml
     /// </summary>
-    public partial class NotificationsControl : UserControl
+    public partial class NotificationsControl
     {
         public NotificationsControl()
         {
             InitializeComponent();
-
-            this.Loaded += NotificationsControl_Loaded;  
-        }
-
-        void NotificationsControl_Loaded(object sender, RoutedEventArgs e)
-        {
-            var window = WpfUtilities.FindUpVisualTree<DynamoView>(this);
-            if (window == null)
-            {
-                return;
-            }
-
-            window.PreviewMouseDown += window_PreviewMouseDown;
-        }
-
-        /// <summary>
-        /// Handle the DynamoView's PreviewMouseDown event.
-        /// When the user click anywhere in the view, clear the ViewModel's warning.
-        /// </summary>
-        /// <param name="sender"></param>
-        /// <param name="e"></param>
-        void window_PreviewMouseDown(object sender, MouseButtonEventArgs e)
-        {
-            var hsvm = (HomeWorkspaceViewModel)((DynamoViewModel)DataContext).HomeSpaceViewModel;
-            hsvm.ClearWarning();
```

Comment:
Do not remove these events. Instead comment the line hsvm.clearwarning. Use the comment - "Commented this after MAGN - 8423 ".